products/sle/main.pm

```diff
@@ -1355,6 +1353,9 @@ elsif (get_var('HPC')) {
             loadtest 'console/hostname';
             loadtest 'hpc/powerman';
         }
+        elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
```

Comment:
actually whole HPC module is supported only starting from SP2 so we can move this check into line 1338